test/libsyncengine/requests/testexclusiontemplatecache.cpp

```diff
@@ -137,29 +165,24 @@ void TestExclusionTemplateCache::tearDown() {
 
 void TestExclusionTemplateCache::testIsExcluded() {
     // Exclude hidden files
-    Parameters params;
-    bool found = false;
-    CPPUNIT_ASSERT(ParmsDb::instance()->selectParameters(params, found) && found);
-    params.setSyncHiddenFiles(false);
-    CPPUNIT_ASSERT(ParmsDb::instance()->updateParameters(params, found) && found);
+    ParametersCache::instance()->parameters().setSyncHiddenFiles(false);
 
     // Test rejected files
     for (const auto &str : rejectedFiles) {
+        std::cout << "Testing rejected item: `" << str << "`\n";
```

Comment:
This is going to clutter the test output. 

Note that is possible to add a message with `CPPUNIT_ASSERT_MESSAGE` if you want to report a problem on a specific object under test.